docs/source-pytorch/accelerators/hpu_basic.rst

```diff
@@ -47,6 +47,40 @@ It uses :class:`~pytorch_lightning.strategies.hpu_parallel.HPUParallelStrategy`
 
 ----
 
+Scale-out on Gaudis
```

Comment:
```suggestion
Run across many machines
```
Or something similar? The term "scale-out" is lesser known imo.

Reply:
I've used it for distributed training in hpu docs, so aligning with the same.